code/modules/reagents/machinery/chem_heater.dm

```diff
@@ -0,0 +1,147 @@
+/obj/machinery/chem_heater
+	name = "chemical heater"
+	density = TRUE
+	anchored = TRUE
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "mixer0b"
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 40
+	//resistance_flags = FIRE_PROOF | ACID_PROOF
```

Comment:
Is this meant to be here?

Reply:
Yeah, in case we ever port resistance_flags and object damage system in general. It's a cool thing I'd like to port one day, but, like many cool things, it's rotting in the backlog nowadays.